Gemfile.jruby-2.5.lock.release

```diff
@@ -575,22 +583,22 @@ GEM
       logstash-codec-plain
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       manticore (>= 0.5.2, < 1.0.0)
-    logstash-mixin-validator_support (1.0.1-java)
+    logstash-mixin-validator_support (1.0.2-java)
       logstash-core (>= 6.8)
-    logstash-output-cloudwatch (3.0.9)
+    logstash-output-cloudwatch (3.0.10)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       logstash-mixin-aws (>= 1.0.0)
-      rufus-scheduler (~> 3.0.9)
+      rufus-scheduler (>= 3.0.9)
     logstash-output-csv (3.0.8)
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       logstash-filter-json
       logstash-input-generator
       logstash-output-file
-    logstash-output-elasticsearch (11.3.1-java)
+    logstash-output-elasticsearch (11.4.1-java)
```

Comment:
:whale2: run this with 
`ruby tools/release/bump_plugin_versions.rb 8.0 8.0.0 patch`
but it seems to have upgraded minor versions ... back to :school: